src/pages/iou/IOUCurrencySelection.js

```diff
@@ -74,6 +76,28 @@ function IOUCurrencySelection(props) {
     const selectedCurrencyCode = (lodashGet(props.route, 'params.currency', props.iou.currency) || CONST.CURRENCY.USD).toUpperCase();
     const iouType = lodashGet(props.route, 'params.iouType', CONST.IOU.MONEY_REQUEST_TYPE.REQUEST);
     const reportID = lodashGet(props.route, 'params.reportID', '');
+    const threadReportID = lodashGet(props.route, 'params.threadReportID', '');
+
+    // Dismiss the modal when the request is paid or deleted
+    useEffect(() => {
```

Comment:
Since this is reused in multiple places, for new request, split, editing request and split, can we update the comment here to describe more thoroughly when we are going to dismiss this and when not?

Reply:
Done - a544459.